packages/schemas/alterations/next-1676823841-update-sie-primary-key.ts

```diff
@@ -0,0 +1,26 @@
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    await pool.query(sql`
+      drop index sign_in_experiences__id;
+      alter table sign_in_experiences
+        drop constraint sign_in_experiences_pkey,
+        add primary key (tenant_id, id);
+    `);
+  },
+  down: async (pool) => {
+    await pool.query(sql`
+      alter table sign_in_experiences
+        drop constraint sign_in_experiences_pkey,
```

Comment:
Should this be `add` instead of `drop`?

Reply:
this is for primary key, add is right below